src/uct/cuda/base/cuda_iface.h

```diff
@@ -50,6 +50,24 @@
     })
 
 
+#define UCT_CUDADRV_CTX_ACTIVE(_state)                             \
+    do {                                                           \
+        CUcontext cur_ctx;                                         \
+        CUdevice dev;                                              \
+        unsigned flags;                                            \
+                                                                   \
+        _state = 0;                                                \
+        /* avoid active state check if no cuda activity */         \
+        if ((CUDA_SUCCESS == cuCtxGetCurrent(&cur_ctx))            \
```

Comment:
typically we write operands like `&&` on the first line, but it is not stated in our coding style though